packages/core/test/unit/config.test.js

```diff
@@ -85,6 +85,21 @@ describe('SnapshotSchema', () => {
     expect(snapshotSchema.$defs.common.properties).toEqual(jasmine.objectContaining({ domTransformation: jasmine.anything() }));
   });
 
+  it('should contain valid dom serialization values', () => {
+    const dom = {
```

Comment:
This you have hardcoded so it will always pass until you remove it from config.
Expectation for test case should be PercyDOM.serialize -> response -> PercyConfig.validate(response) -> No errors